tests/Microsoft.Identity.Test.Android.UIAutomation/Microsoft.Identity.Test.Android.UIAutomation.csproj

```diff
@@ -29,6 +29,11 @@
     <Reference Include="System" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers">
```

Comment:
Why don't these projects pick up Directory.Build.props ? We should not duplicate this code.